mongo/dataset_test.go

```diff
@@ -218,6 +228,7 @@ func TestDatasetUpdateQuery(t *testing.T) {
 			"next.canonical_topic":          canonicalTopic,
 			"next.subtopics":                subtopics,
 			"next.survey":                   survey,
+			"next.related_content":			 relatedContent,
```

Comment:
The relatedContent variable is not quite aligned. If you do a 'Go Tools' > 'Go Fmt Project', in GoLand, then it will fix this.